students/alex_whitty/session08/test_circle.py

```diff
@@ -0,0 +1,85 @@
+from math import pi
+from circle import Circle, Sphere
+
+
+def test_radius():
+    c = Circle(2)
+    assert c.radius == 2
+
+
+def test_diameter():
+    c = Circle(3)
+    assert c.diameter == 6
+
+
+def test_set_diameter():
+    c = Circle(4)
+    c.diameter = 5
+    assert c.diameter == 5
+    assert c.radius == 2.5
+
+
+def test_area():
+    c = Circle(6)
+    assert c.area == pi * 36
+
+def test_alt_construct():
+    c1 = Circle.from_diameter(10)
+    assert c1.diameter == 10
+
+
+def test_from_diameter():
+    c = Circle.from_diameter(8)
+    assert c.diameter == 8
+    assert c.radius == 4
+
+
+def test_string():
+    c = Circle(9)
+    assert str(c) == "Circle with radius: 9"
+
+def test_string_repr():
+    c = Circle(12)
+    assert repr(c) == "Circle(12)"
+
+
+def test_less():
+    c1 = Circle(2)
+    c2 = Circle(5)
+    assert (c1 < c2) is True
```

Comment:
when using `assert` you'll want to just use the operators, no need for the `is True`